esphome/components/rtttl/rtttl.h

```diff
@@ -60,6 +67,7 @@ class Rtttl : public Component {
   uint16_t note_duration_;
 
   uint32_t output_freq_;
+  double gain_{0.5};
```

Comment:
We try to use floats instead of doubles
```suggestion
  float gain_{0.5f};
```